packages/web/src/pages/desktop-app/desktop-app.tsx

```diff
@@ -0,0 +1,141 @@
+import { DownloadOutlined } from "@ant-design/icons";
+import { Button, Col, Image, Row } from "antd";
+import Link from "antd/lib/typography/Link";
+import Title from "antd/lib/typography/Title";
+import React from "react";
+import AppVersionFile from "@coh2ladders/web/public/electron-app-version.json";
+
+const DesktopApp: React.FC = () => {
+  return (
+    <>
+      <Row justify="center" style={{ paddingTop: "20px" }}>
+        <Col xs={23} xxl={17}>
```

Comment:
FYI @JohannesMerkt  I've tried to write down few tips and tricks - for how I solved this:

So first I have created one master <Row> and <Col> and put additional <Row> inside them.  You can have grids inside grids. 

That way I can have only one central col, where I can setup the xs and xxl styling. I've setup 23. You rarely want to have 24 as that's very close to the border - on most phones will cause problems. 

Also when you look lower in the code I am again using col 24 because, you can imagine that with xxl 17 , the page will be split into 17/24 ~70% and that by inserting a new row and another col I can again split by 24. 

In the cols which are next to each other I am using for example 8 and 16 on xl. But you need to count with mobile phones. On those you have anything which is next to each other displayed below , so that's why I am adding 24. So it will take whole page. (But there is gonna still be border - because there is master col with 23). 

Now it should work well on any display. 
![image](https://user-images.githubusercontent.com/8086995/154164908-8e8ed73d-d72a-4bfe-91c9-f1582eb67a71.png)

![image](https://user-images.githubusercontent.com/8086995/154164948-128ad8b8-efdb-4746-a8ce-f358236867d7.png)

![image](https://user-images.githubusercontent.com/8086995/154164988-62f7b86b-edef-4ffa-b578-e8a2e9e8969e.png)

There is one thing which I didn't solve correctly. When the text is on the right side on the mobile screen it will first display image and than text. I am really no sure how to solve it right now, I think that you want to have the html of the text to be before the image, but put there floating, align right, maybe offset with column.

But I don't think it's that crucial to fix. I think it looks good regardless
![image](https://user-images.githubusercontent.com/8086995/154165190-7f2ede1c-30bf-4994-9d94-6dc021ef2e49.png)

Reply:
Wow impressive. I will have a look at the code to fully understand how you solved it!